### PR TITLE
Don't do JSON as a database, kids.

### PR DESCRIPTION
**Pull request by Xithrius**
It won't do you well.

Finishes off one of the check marks for #695.

### CODE REVIEW

**Review by jchristgit — APPROVED** [review]
This migrates the pin, so I'm approving.

That said, I do think the content is a bit weird, since the second paragraph makes the following points:

- "No redundancy built into JSON"
-  "No built in ways to keep it safe and backed up"
- "No built in encryption", with some inclusion of GDPR

This, to me, sounds like standard databases offer any of that, out of the box. Unfortunately there's no magic setting or function in PostgreSQL where I can go `SELECT spin_up_slaves(3)`, `SELECT perform_encrypted_redundant_backup()` and `SELECT pg_encrypt_harddrive()`, those are all still left to the operator.

Regardless, this makes some good points, so let's get it in.

**Comment by jchristgit** [comment]
Let's maybe revise the content later, but migrate the pin here first.

File: pydis_site/apps/content/resources/guides/python-guides/why-not-json-as-database.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+JSON, unlike relational databases, has no way to store relational data, which is a very commonly needed way of storing data. Relational data, as the name may suggest, is data that relates to other data. For example if you have a table of users and a table of servers, the server table will probably have an owner field, where you'd reference a user from the users table. [This is only relevant for relational data]
+
+JSON is primarily a KV (key-value) format, for example {"a":"b"} where a is the key and b is the value, but what if you want to search not by that key but by a sub-key? Well, instead of being able to quickly use var[key], which in a Python dictionary has a constant return time (for more info look up hash tables), you now have to iterate through every object in the dictionary and compare to find what you're looking for. Most relational database systems, like MySQL, MariaDB, and PostgreSQL have ways of indexing secondary fields apart from the primary key so that you can easily search by multiple attributes.
```
Comment (Bluenix2) [inline pydis_site/apps/content/resources/guides/python-guides/why-not-json-as-database.md]: Yeah I still think these should be split into multiples lines capping at like 120 which is what Python Discord uses for code

**Review by Bluenix2 — APPROVED** [review]
That was my only complaint. Thanks for the PR Xithrius (and Mina for finishing it 💯)!

**Pull request merged**